spec/views/shared/_zero_results.html.erb_spec.rb

```diff
@@ -24,8 +25,16 @@
 
   it "should render text indicating tips to modify the search along with links to the relevant search" do
     render
-    expect(rendered).to have_css("li", text: /Your search:/)
-    expect(rendered).to have_css("a", text: /Title:/)
+    expect(rendered).to have_css("dt", text: /Your current search/)
+    expect(rendered).to have_css("a", text: /Title >/)
+    expect(rendered).to have_css("dt", text: 'Remove limit(s)')
+    expect(rendered).to have_css("dt", text: /Search all fields/)
+  end
+
+  it "should render chat and search tools" do
```

Comment:
Let's not use "should".  That says "it may do this, but it should do something else" rather let's assert what it does:  `it 'renders chat and search tools'`

Reply:
✅ thanks! I've updated the other specs in the file to match.